README.md

```diff
@@ -247,7 +247,7 @@ Horizontal stack with :
       color: rgb(223, 255, 97)
       icon: mdi:volume-plus
       tap_action:
-        action: call-service
+        action: service
```

Comment:
This should still be `call-service`

Reply:
Right, that's strange... I don't remember changing that 😅